securedrop/requirements/update_python_dependencies

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Usage: ./update-python-dependencies
```

Comment:
Since you're changing the name of `pip-update.sh`, these docs still need updating: https://docs.securedrop.org/en/latest/development/tips_and_tricks.html#upgrading-or-adding-python-dependencies